kolibri/core/content/api.py

```diff
@@ -861,13 +860,8 @@ class ContentNodeViewset(InternalContentNodeMixin, RemoteMixin, ReadOnlyValuesVi
     pagination_class = OptionalContentNodePagination
 
     def retrieve(self, request, pk=None):
-
-        try:
```

Comment:
Hi @manzil-infinity180! We still want this functionality of checking if the pk is a valid UUID or not. We want both of the checks that if the UUID is invalid then we throw a 400 and if it is None then we would raise 404. so both the checks should simultaneously exist.